src/java/org/apache/cassandra/gms/FailureDetector.java

```diff
@@ -107,20 +107,35 @@ private static long getInitialValue()
 
     public String getAllEndpointStates()
     {
-        return getAllEndpointStates(false);
+        return getAllEndpointStates(false, false);
+    }
+
+    public String getAllEndpointStatesWithResolveIp()
```

Comment:
should not it be "...WithResolvedIp()" ? Notice "Resolved" instead of "Resolve". This change should be applicable everywhere in this patch.

Reply:
@smiklosovic ,
Thanks for the quick review!
It's my first open source merge request, I was really nervous if my implementation was even in the right direction 😅 

By the way, I am not resolving any thread by myself.

Please tell me if it's ok to just leave a comment (and commit) and wait for a reviewer to resolve?
Or should I resolve a thread to which I have attached the commit with the fix?